gov.pnnl.goss.gridappsd/bnd.bnd

```diff
@@ -29,7 +29,7 @@
 	osgi.enroute.base.api,\
 	org.mockito.mockito-all,\
 	httpcore,\
-	blazegraph.cim2glm;version=19.1.1,\
+	cimhub.lib;version=0.2.1,\
```

Comment:
I thought you said you were releasing 0.0.1 version?

Reply:
yeah, I had to make some changes because there were some errors when testing 0.0.1 with GridAPPS-D. With those changes bndtool increased the version.